src/sbin/sympa.pl.in

```diff
@@ -179,6 +181,14 @@ if ($options{'add'}) {
     exit 0;
 } elsif ($options{'version'}) {
     Sympa::CLI->run({}, 'version');
+} elsif ($options{'review'}) {
+    Sympa::CLI->run(\%options, 'review', $options{review});
+} elsif ($options{'reset_members_bounces'}) {
+    Sympa::CLI->run({}, 'bouncers', 'reset',
+        $options{'reset_members_bounces'});
+} elsif ($options{'del_bounced_members'}) {
+    Sympa::CLI->run({}, 'bouncers', 'del',
+        $options{'del_bounced_members'});
```

Comment:
I personally think that the new features need not to support the old-style command line format.

Reply:
Ok with that (die, old syntax, die! :sweat_smile: ). I’ll update the PR.